README.md

```diff
@@ -68,6 +68,12 @@ In addition you would need
     node authentication_sample.js
     ```
```

Comment:
In the beginning of this file, we have

> The Key Vault client is then used to authenticate to the vault and set/retrieve a sample secret.

This should be changed to

> The Key Vault secret client is then used to authenticate to the vault and set/retrieve a sample secret.

Because we now have different clients for keys, secrets and certificates

In the Quick Start section, the first step is to install node. We don't need this because the previous section on prerequisites already covers this